repairnator/repairnator-realtime/src/test/java/fr/inria/spirals/repairnator/realtime/TestRTScanner.java

```diff
@@ -74,4 +74,15 @@ public void testRepositoryWithoutJavaLanguageIsNotInteresting() {
         boolean result = rtScanner.isRepositoryInteresting(repositoryOptional.get().getId());
         assertFalse(result);
     }
+
+    /**
+     * Note this test might fail locally if you don't have activeMQ
+     * In that case this test can be temporarily be commented out
+     */
+    @Test
+    public void testActiveMQRunnerConnection()
```

Comment:
good start! another test that is required is to test that Repairnator actually pushes something to the queue.

Reply:
Actually this testconnection does both trying to connect to the queue and also send a "Test message" string to that queue and see if it cause any exception :). So it does push something to the queue.

Reply:
I mean, in the test, the rtscanner should analyze a build and sends a message to the queue. An assertion would check that the buildid is well received on the queue.

Reply:
Ah I see, It is fixed :) in the commit "Fix: add integration test for ActiveMQPipelineRunner".